tests/test_sdiff.py

```diff
@@ -13,7 +13,7 @@ def _load_fixture(*path):
 
 def _read_test_files(dirpath):
     path = Path(os.path.join('tests/fixtures', dirpath))
-    filenames = set(f.name.split('.')[0] for f in path.glob('*.md'))
+    filenames = {f.name.split('.')[0] for f in path.glob('*.md')}
```

Comment:
changing `set`  to `dict` was intended?

Reply:
this is how `pyupgrade --py36-plus` works by default. `{}` can create dict if `key: value` is used or set if `value` is used as item. TBH I prefer set() as constructor but whatever..

Reply:
ah, pardon that's set 😅